SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/DuoSamplesListFragment.kt

```diff
@@ -0,0 +1,150 @@
+import com.microsoft.device.display.samples.twopage.TwoPageActivity
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+class DuoSamplesListFragment : Fragment() {
+
+    companion object {
+        private const val GIT_LINK_DRAG_AND_DROP = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/DragAndDrop"
+        private const val GIT_LINK_DUAL_VIEW = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/DualView"
+        private const val GIT_LINK_HINGE_ANGLE = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/HingeAngle"
+        private const val GIT_LINK_LIST_DETAILS = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/HingeAngle"
+        private const val GIT_LINK_PEN_EVENTS = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/PenEvents"
+        private const val GIT_LINK_TWO_PAGE = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/TwoPage"
+        private const val GIT_LINK_COMPANION_PANE = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/CompanionPane"
+        private const val GIT_LINK_EXTENDED_CANVAS = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/ExtendCanvas"
+        private const val GIT_LINK_SECOND_SCREEN = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/IntentToSecondScreen"
+        private const val GIT_LINK_QUALIFIERS = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/Qualifiers"
+        private const val GIT_LINK_MULTIPLE_INSTANCES = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/MultipleInstances"
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.duo_samples_fragment_first, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<Button>(R.id.button_1_drag_and_drop_try).setOnClickListener {
+            val intent = Intent(requireActivity(), DragAndDropActivity::class.java)
+            startActivity(intent)
+        }
+        view.findViewById<View>(R.id.link_1_dear_and_drop).setOnClickListener {
+            openGitLink(GIT_LINK_DRAG_AND_DROP)
+        }
+
+        view.findViewById<Button>(R.id.button_2_dual_view).setOnClickListener {
+            val intent = Intent(requireActivity(), DualViewActivity::class.java)
+            startActivity(intent)
+        }
+        view.findViewById<View>(R.id.link_2_dual_view).setOnClickListener {
+            openGitLink(GIT_LINK_DUAL_VIEW)
+        }
+
+        view.findViewById<Button>(R.id.button_3_hinge_angle).setOnClickListener {
+            val intent = Intent(requireActivity(), HingeAngleActivity::class.java)
```

Comment:
You should create a method openActivity(activityKlass) or openScreen(activityKlass) or openSample(activityKlass) and use it.